### ISSUE TITLE
Tradition is a component of the spellcaster, not the spells.

### ISSUE THREAD

**Issue by Minalien** [superseva/ac2d20#17]
As the system is currently structured, tradition is set on individual spells. This isn't accurate to the game mechanics; while certain player-facing spells are usually provided to player characters as part of a certain type (e.g. Celtic spells that players take will almost definitely be used by a Tradition-based spellcaster), at the end of the day the type of casting comes down to the PC or NPC rather than the spell itself.

This is especially important when setting up NPC spells and spellcasters; even just within the GMG, NPCs can use many of the same spells (mostly from lovecraftian gods' spell lists) with different traditions. So setting up the NPCs' stat blocks means adding the spell, then re-editing each spell on the character's sheet in order to make sure it has the appropriate attribute selected.

TLDR: Spellcasting type (& therefore the attribute used when casting) should be specified on PC and NPC sheets, not on the spells. I can dig up rulebook page numbers to reference if needed.

**Comment by superseva** [comment]
Hi, I agree with this but unfortunately I am putting soft brakes on  the maintenance for a while. I simply don’t have time to do it atm. 
If someone makes a push request I would gladly merge it. This change shouldn’t be hard to implement.

**Comment by superseva** [comment]
Done and done .
v10.0.4
Didn't have time to migrate actors but I think it should be fine
